autoscheduler/frontend/src/components/InfoPage/InfoPage.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import * as ReactMarkdown from 'react-markdown';
+import { RouteComponentProps } from '@reach/router';
+import { Paper } from '@material-ui/core';
+import CaptionedGif from './CaptionedGif/CaptionedGif';
+import * as styles from './InfoPage.css';
+
+const headers = `# How To Use
+---`;
+
+const quickStartTitle = '## Quick Start Guide';
+
+const quickStartList = ` * To begin, add a course by clicking the add course button and typing 
+out the course you want. Repeat this for every course you want to add to your schedule. During registration, 
+this information is updated about every 15 minutes.
+
+* If you do not want classes at a certain time, click and drag on the calendar
+to block off those &lsquo;&lsquo;busy times&rsquo;&rsquo; when generating schedules.
+To remove your busy times, you can click the trash icon. To quickly remove them in
+groups, you can click and hold on an edge then drag to rearrange multiple blocks so
+that all your times are on the same day before clicking the icon.
+
+* Hit the generate schedules button to generate up to 5 schedules with the options
+you selected. New schedules will overwrite all previous unsaved schedules.
+If you see a schedule you like, click the lock icon to save it.
+The icon should change so that the lock is now closed. You can also
+give the schedule a name to identify it easier by clicking the pencil icon.
+
+* You can click on a generated schedule to make it appear on the big calendar.
+You can also click the details button to open a pop up with more information
+about the sections in that schedule.
+
+* Whenever you click the generate button, only up to five possible schedules will be
+generated. This is because some courses have a lot of sections and showing every single
+possible combination at once can be be very overwhelming. If you are not satisfied
+with any of the generated schedules, hitting the button again will automatically
+remove all unsaved schedules and generate up to five more new ones.`;
```

Comment:
```suggestion
const quickStartList = `
1. To begin, add a course by clicking the add course button and typing 
out the course you want. Repeat this for every course you want to add to your schedule.

2. Click and drag on the calendar to mark your unavailable (&quot;busy&quot;) times, and we
will generate schedules without classes at these times (reword).
To remove your busy times, you can click the trash icon. 

3. Hit the generate schedules button to generate up to 5 schedules with the options
you selected. New schedules will overwrite all previous unsaved schedules.
If you see a schedule you like, click the lock icon to save it.

4. You can click on a generated schedule to make it appear on the right calendar.
You can also click the details button to open a pop up with more information
about the sections in that schedule.

5. Whenever you click the generate button, only up to five possible schedules will be
generated. If you are not satisfied with any of the generated schedules, hitting the button
again will automatically remove all unsaved schedules and generate up to five more possible combinations.`;

```
My main thing with this is that since it's the "quick start" guide, each instruction should be very short and to the point. As such, I've tried to simplify these as much as possible. Also changed them to be numbered.

Note that I removed the `During registration, this information is updated about every 15 minutes.` I still think we should include this, just not right here (maybe in additional tips/info?).

For point 2: Not sure if we should call them unavailable or busy times (probably busy though). Also wasn't sure how to word it, but figured we should mention something like we generate courses around these time.

For point 5: I think this is necessary, but I'm not sure exactly how to word it (but I would like it to be shorter). The main things we want to say is that generate schedules only generates a subset of combinations, and that these combinations are "random"

Reply:
I moved the updated every 15 minutes tip to the additional tips section for now.

For point 2, I like using "busy" better than "unavailable" as well. However, as we don't refer to them ever again (except for the multi-remove tip in additional tips), giving them a shorter name seems unnecessary. For this reason I removed "busy".

For point 5, I just added the contents to point 3 and then removed it. Although it makes point 3 longer, I think the changes still leave it very to the point and the information flows better when grouped here.

Reply:
Could probably remove a bit more from point 3 by not repeating up to 5 again the second time for when they click the button again